R/summarize.R

```diff
@@ -10,7 +10,7 @@
 # suppressPackageStartupMessages(library(rlang))
 # conflicted::conflict_prefer("filter", "dplyr")
 
-#' Filter by Domains
+#' filterByDomains
```

Comment:
for consistency, modified the titles of function documentation to the function call.